tmt/steps/prepare/ansible.py

```diff
@@ -1,7 +1,25 @@
+import sys
+from typing import Any, List, Optional
+
 import click
 
 import tmt
 
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+# Type annotation for "data" package describing a guest instance. Passed
+# between load() and save() calls.
+DataType = TypedDict(
```

Comment:
I'd probably pick a more descriptive name. "Data" can mean many things, in my Artemis plugin, I went with `StepDataType`, to reduce the number of possible "data"-like bundles a bit.

Reply:
looks resolved to me

Reply:
@guy9050 pls resolve those which you feel you resolved them